core/src/main/java/org/apache/accumulo/core/client/ClientSideIteratorScanner.java

```diff
@@ -62,9 +62,24 @@
   private long readaheadThreshold = Constants.SCANNER_DEFAULT_READAHEAD_THRESHOLD;
 
   /**
+   * @deprecated since 1.7.0 was never intended for public use.  However this could have been used by anything extending this class.
+   */
+  @Deprecated
+  public class ScannerTranslator extends ScannerTranslatorImpl {
+    public ScannerTranslator(Scanner scanner) {
+      super(scanner);
+    }
+
+    @Override
+    public SortedKeyValueIterator<Key,Value> deepCopy(final IteratorEnvironment env) {
+      return new ScannerTranslator(scanner);
+    }
+  }
+
+  /**
    * A class that wraps a Scanner in a SortedKeyValueIterator so that other accumulo iterators can use it as a source.
    */
-  public class ScannerTranslator implements SortedKeyValueIterator<Key,Value> {
+  private class ScannerTranslatorImpl implements SortedKeyValueIterator<Key,Value> {
```

Comment:
No problem.  I appreciate everyone looking at this.  I know its painful to review.